gogio/build_info.go

```diff
@@ -94,6 +94,10 @@ func (s Semver) String() string {
 	return fmt.Sprintf("%d.%d.%d.%d", s.Major, s.Minor, s.Patch, s.VersionCode)
 }
 
+func (s Semver) StringCompact() string {
+	return fmt.Sprintf("%d.%d.%d%d", s.Major, s.Minor, s.Patch, s.VersionCode)
```

Comment:
That won't generate strictly increasing versions, will it? Say patch == 1, version code ==  1 for release A, then patch == 0 and version code == 2 for a later release B.

Putting patch last may work, or maybe just throw it away?

Reply:
I drop the version code, I also add a comment to make it clear why it happens.